frontend/src/components/createGroupWindow.jsx

```diff
@@ -0,0 +1,78 @@
+import Button from "./button";
+import InputBox from "./inputBox";
+import ModalHeader from "./modalHeader.jsx";
+import { useHistory } from "react-router-dom";
+import { useState } from "react";
+import ModalWindow from "./modalWindow";
+import "../css/personalPage.css";
+
+function CreateGroupWindow({ trigger, setTrigger, groups, setGroups }) {
+    const history = useHistory();
+    const [group_name, setGroupName] = useState("");
+    const [description, setGroupDescription] = useState("");
+
+    const attemptCreatGroup = async () => {
+        if (group_name === "" || description === "") {
+            alert("Please enter group name and group description.");
+            return;
+        }
+
+        const response = await fetch("http://localhost:3000/v1/groups/", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                Authorization:
+                    "bearer " + window.localStorage.getItem("authToken"),
+            },
+            body: JSON.stringify({
+                group_name,
+                description,
+            }),
+        });
+
+        const responseJSON = await response.json();
+        if (responseJSON.group_id) {
+            setGroups([...groups, { group_name }]);
+            setTrigger(false);
+            history.push("/my_notes");
```

Comment:
I think we should redirect to the newly created group page

Reply:
We can stay in my note page so far.

Reply:
don't need to push to any page since we are staying on the same one